src/app/shared/helpers/url.ts

```diff
@@ -93,3 +94,18 @@ export function openNewTab(href: string, currentLang?: LanguageConfig): void {
   window.open(formatUrl(href, currentLang), '_blank');
 }
 
+/**
+ * Return the value of the given field as a boolean (or null if not set)
+ * Interprets any value as true except "0" (if `boolToQueryParamValue` was used) or "false" (in case the boolean value was used directly
+ * in url)
+ */
+export function boolFromParamMap(paramMap: ParamMap, field: string): boolean|null {
+  const val = paramMap.get(field);
+  if (val === null) return null;
+  if (val === '0' || val === 'false') return false;
+  return true;
+}
```

Comment:
It restricts usage to `paramMap` only, while we can also use `params` as an object and not a map with `this.activatedRoute.snapshot.queryParams`.

A `queryParamValueToBool(value: string | undefined | null): boolean | null` would be more flexible (or `boolFromQueryParamValue`).

Plus, I'm not convinced by the "false" handling, I think we should stick to `0` and `1`

Reply:
fixed